ff_bot/ff_bot.py

```diff
@@ -299,7 +305,9 @@ def bot_main(function):
     elif function=="get_trophies":
         text = get_trophies(league)
     elif function=="get_final":
-        text = "Final " + get_scoreboard_short(league)
+        # on Tuesday we need to get the scores of last week
+        week = league.current_week - 1
+        text = "Final " + get_scoreboard_short(league, week=week)
         text = text + "\n\n" + get_trophies(league)
```

Comment:
Would the get_trophies also need to be `get_trophies(league, week=week)` since the league's current_week is the upcoming week?

Reply:
It wouldn't, all of the trophies are calculated off of the box score, which can accept a None type for a week.